### PR TITLE
feat: add option for default modifier for v10 types

### PR DESCRIPTION
**Pull request by johannessjoberg**
If you create your contentful client with an option like `contentful.createClient({ ... }).withoutUnresolvableLinks` and reuse it everywhere, it would improve DX to be able to use `TypeTest` directly instead of explicitly stating `TypeTest<'WITHOUT_UNRESOLVABLE_LINKS'>` or using `TypeTestWithoutUnresolvableLinksResponse`.

This PR adds default modifier option to achieve that.

### CODE REVIEW

**Comment by johannessjoberg** [comment]
@marcolink What are your thoughts on these changes?

**Comment by johannessjoberg** [comment]
Bump

**Comment by zkeetonev** [comment]
@marcolink Would love to see this get merged as it doesn't make sense to repeat this for every usage of the generated types.

@johannessjoberg Should `WITH_ALL_LOCALES` be added as an option as well? I would also like it if no value is provided to the `--modifier` flag, then it could just use undefined as a default (`Modifiers extends ChainModifiers = undefined`).

**Review by marcolink — COMMENTED** [review]
Hi 👋 

I have added a comment. Overall this looks straight forward and valuable 👍

File: src/commands/index.ts [hunk 1]
```diff
@@ -31,6 +31,7 @@ class ContentfulMdg extends Command {
     jsdoc: Flags.boolean({ char: 'd', description: 'add JSDoc comments' }),
     typeguard: Flags.boolean({ char: 'g', description: 'add type guards' }),
     response: Flags.boolean({ char: 'r', description: 'add response types' }),
+    modifier: Flags.string({ char: 'm', description: 'add default modifier (v10 only)' }),
```
Comment (marcolink) [inline src/commands/index.ts:34]: Should we:
- allow all possible modifiers
- allow an array of modifiers to also support combinations?

**Comment by marcolink** [comment]
cc @johannessjoberg

**Comment by andreaspalsson** [comment]
> TL;DR I have a branch with my suggested way forward that I will push if there is no action on this PR in the next days!

@marcolink Please feel free to push it. I don't think we will have the bandwidth to update this with support for more modifiers in the coming weeks.
